### ISSUE TITLE
MakeDataCountApiIT.testMakeDataCountGetMetric IT failure

### ISSUE THREAD

**Issue by pameyer** [IQSS/dataverse#5662]
In docker-aio (2x container, 2x run) on 943931363331095b831edabf697e1b26243c0e9d :

```
[ERROR] Failures: 
[ERROR]   MakeDataCountApiIT.testMakeDataCountGetMetric:60 Expected status code <200> doesn't match actual status code <400>.

[INFO] 
[ERROR] Tests run: 85, Failures: 1, Errors: 0, Skipped: 2
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------

```

```
-------------------------------------------------------------------------------
Test set: MakeDataCountApiIT
-------------------------------------------------------------------------------
Tests run: 3, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 1.089 s <<< FAILURE! - in MakeDataCountApiIT
testMakeDataCountGetMetric  Time elapsed: 1.082 s  <<< FAILURE!
java.lang.AssertionError: 
Expected status code <200> doesn't match actual status code <400>.

        at edu.harvard.iq.dataverse.api.MakeDataCountApiIT.testMakeDataCountGetMetric(MakeDataCountApiIT.java:60)

```

**Comment by sekmiller** [comment]
This is blocked by #5649 - PostgreSQL 9.3 is required, 9.6 strongly recommended

**Comment by pdurbin** [comment]
The test passes for me too. I'm on 3c836a980 on develop.

@pameyer could the "copy file" stuff in the screenshot below be a problem?

![Screen Shot 2019-03-22 at 3 41 10 PM](https://user-images.githubusercontent.com/21006/54848930-00f18600-4cb9-11e9-8324-0a757b22bc86.png)

**Comment by pameyer** [comment]
@pdurbin not sure - I don't think `sushi_sample_logs.json` is getting into the container; but it doesn't look to me like the pheonix deployment scripts do anything with it either.  Which suggests it doesn't need to be present server-side to run the tests, but my knowledge of the moving parts in mdc means I probably shouldn't rule it out.

**Comment by pameyer** [comment]
Rerun on 3c836a9801b9c21f888d556d85f6db481d025bf8 ; MakeDataCountApiIT.testMakeDataCountGetMetric still failing (will take a look to see if fiddling w\ sushi sample helps).

Now also seeing 13 errors:
``
[ERROR] Errors: 
[ERROR]   FilesIT.testAccessFacet:1030 ? FileNotFound src/main/webapp/resources/images/f...
[ERROR]   FilesIT.testAddFileBadJson:242 ? FileNotFound src/main/webapp/resources/images...
[ERROR]   FilesIT.testReplaceFileBadJson:820 ? FileNotFound src/main/webapp/resources/im...
[ERROR]   FilesIT.testRestrictAddedFile:995 ? FileNotFound src/main/webapp/resources/ima...
[ERROR]   FilesIT.testRestrictFile:920 ? FileNotFound src/main/webapp/resources/images/f...
[ERROR]   FilesIT.test_001_AddFileGood:115 ? FileNotFound src/main/webapp/resources/imag...
[ERROR]   FilesIT.test_002_AddFileBadDatasetId:167 ? FileNotFound src/main/webapp/resour...
[ERROR]   FilesIT.test_003_AddFileNonExistentDatasetId:188 ? FileNotFound src/main/webap...
[ERROR]   FilesIT.test_004_AddFileBadToken:214 ? FileNotFound src/main/webapp/resources/...
[ERROR]   FilesIT.test_005_AddFileBadPermissions:274 ? FileNotFound src/main/webapp/reso...
[ERROR]   FilesIT.test_007_ReplaceFileUnpublishedAndBadIds:649 ? FileNotFound src/main/w...
[ERROR]   FilesIT.test_008_ReplaceFileAlreadyDeleted:736 ? FileNotFound src/main/webapp/...
[ERROR]   UsersIT.testMergeAccounts:194 ? FileNotFound src/main/webapp/resources/images/...
``

``
grep FileNotFound target/surefire-reports/FilesIT.txt 
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
java.io.FileNotFoundException: src/main/webapp/resources/images/favicondataverse.png (No such file or directory)
``

**Comment by pameyer** [comment]
copying `./doc/sphinx_bootstrap_theme/bootstrap/static/images/favicondataverse.png` to `./src/main/webapp/resources/images/favicondataverse.png` fixes this.  So either the tests are referring to the favicon from the wrong location, or it needs to be in the resource manager and isn't.

**Comment by pameyer** [comment]
@pdurbin good find - putting `/tmp/sushi_sample_logs.json` on the same system (and the favicon fix) running glassfish has everything passing.

**Comment by pdurbin** [comment]
@pameyer for testing in docker-aio, isn't this issue a bit blocked by #5374? I know you have some workarounds but from my perspective, that other issue is about how docker-aio doesn't "just work" the way it used it.

**Comment by pameyer** [comment]
@pdurbin #5374 adds a step; but I don't think it blocks the `testMakeDataCountGetMetric` failure.  And the `FilesIT` errors should be the same regardless of source repo / glassfish server configuration.

**Comment by pdurbin** [comment]
@pameyer ok, can you please add the extra manual step here? I just looked through #5374 again but I'm confused what the step is. It looks like over at https://github.com/IQSS/dataverse/issues/5374#issuecomment-457725574 I reported that I ran...

    docker exec -it dv /usr/local/glassfish4/bin/asadmin start-domain

... but then Glassfish went immediately down again.

**Comment by pameyer** [comment]
Short version - skip `prep_it.bash` and fall back to `docker build ..` , `docker run ...` , connect to the container `docker exec -it dv bash` , and run `setupIT.bash` from inside.  I still don't know what's different, but that approach doesn't end up killing glassfish.

**Comment by djbrooke** [comment]
Closed in favor of #5686, which includes this is in the list of test failures.
